packages/solid-crs-manage/lib/app.ts

```diff
@@ -12,9 +12,13 @@ import './index';
  *
  * https://github.com/davidkpiano/xstate/tree/master/packages/xstate-inspect
  */
-inspect({
-  iframe: false, // open in new window
-});
+if (process.env.MODE === 'DEV') {
+
+  inspect({
+    iframe: false, // open in new window
+  });
+
+}
```

Comment:
Shorter?
```suggestion
inspect({
  iframe: (process.env.MODE !== 'DEV'), // open in new window
});
```

Reply:
I think it's the whole `inspect` thing that should be skipped when mode is `DEV`. `iframe` just decides whether it should open in a new window or not.